### PR TITLE
update TG-Watch to match cp main

### PR DESCRIPTION
**Pull request by TG-Techie**
drop the 02a/02b suffix.

### CODE REVIEW

**Comment by makermelissa** [comment]
Thanks! This should probably wait until the next release of CP to be merged or it will appear as an unknown board.

**Review by makermelissa — APPROVED** [review]
Seems both are there at this point, so no harm in merging this.

**Comment by TG-Techie** [comment]
Thank you!

**Pull request merged**
